src/components/screens/main-screen.tsx

```diff
@@ -199,7 +294,7 @@ const MainScreen: React.FC = () => {
    * Renders search
    */
   const renderSearch = () => (
-    <PaperCard elevation={6} sx={{ width: 650 }}>
+    <PaperCard elevation={6} sx={{ width: 700 }}>
```

Comment:
Why is this width changed

Reply:
To make it the same width as the courses. I thought it looked cleaner that way (visually). What do you think?

Reply:
I wanted to have different widths to make the search box visually distinguished. But since you have changed it, I'll leave it